src/bun.js/test/jest.zig

```diff
@@ -2003,6 +2040,13 @@ inline fn createScope(
                     tag_to_use = .skipped_because_label;
                 }
             }
+
+            if (runner.test_options.test_line_filters.count() > 0) {
+                if (!runner.matchesLineFilter(line_number, parent)) {
```

Comment:
this should be abstracted into a single method that does the lookup instead of doing both of these checks.